Framework/Algorithms/inc/MantidAlgorithms/ReflectometryReductionOneAuto3.h

```diff
@@ -91,31 +79,34 @@ class DLLExport ReflectometryReductionOneAuto3
   /// Rebin and scale a workspace in Q
   Mantid::API::MatrixWorkspace_sptr
   rebinAndScale(Mantid::API::MatrixWorkspace_sptr inputWS,
-                RebinParams const &params);
+                const RebinParams &params);
   /// Crop a workspace in Q
   MatrixWorkspace_sptr cropQ(MatrixWorkspace_sptr inputWS,
-                             RebinParams const &params);
+                             const RebinParams &params);
   /// Populate algorithmic correction properties
   void populateAlgorithmicCorrectionProperties(
       Mantid::API::IAlgorithm_sptr alg,
       Mantid::Geometry::Instrument_const_sptr instrument);
   /// Get a polarization efficiencies workspace.
   std::tuple<API::MatrixWorkspace_sptr, std::string, std::string>
   getPolarizationEfficiencies();
-  void applyPolarizationCorrection(std::string const &outputIvsLam);
+  void applyPolarizationCorrection(const std::string &outputIvsLam);
   API::MatrixWorkspace_sptr getFloodWorkspace();
-  void applyFloodCorrection(API::MatrixWorkspace_sptr const &flood,
+  void applyFloodCorrection(const API::MatrixWorkspace_sptr &flood,
                             const std::string &propertyName);
   void applyFloodCorrections();
   double getPropertyOrDefault(const std::string &propertyName,
                               const double defaultValue, bool &isDefault);
-  void setOutputWorkspaces(WorkspaceNames const &outputGroupNames,
+  void setOutputWorkspaces(const WorkspaceNames &outputGroupNames,
                            std::vector<std::string> &IvsLamGroup,
                            std::vector<std::string> &IvsQBinnedGroup,
                            std::vector<std::string> &IvsQGroup);
   WorkspaceNames getOutputNamesForGroups(const std::string &inputName,
                                          const std::string &runNumber,
                                          const size_t wsGroupNumber);
+  void getTransRun(std::map<std::string, std::string> &results,
```

Comment:
```suggestion
  void getTransmissionRun(std::map<std::string, std::string> &results,
```

There is no real need to abbreviate this here.